### PR TITLE
Warning for the installation instruction on README

### PR DESCRIPTION
**Pull request by 1ucian0**
Screenshot 2024-03-16 at 13 54 54

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,6 +20,9 @@ For more details on how to use Qiskit, refer to the documentation located here:
 
 ## Installation
 
+| ⚠️  Do not try to upgrade an existing Qiskit 0.* environment to Qiskit 1.0 in-place. [Read more](https://docs.quantum.ibm.com/api/migration-guides/qiskit-1.0-installation).| 
+|---- |
```
Comment (jakelishman) [inline README.md]: You might want to try [the GitHub call-out boxes syntax](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#alerts) for this?

```suggestion
> [!WARNING]
> Do not try to upgrade an existing Qiskit 0.* environment to Qiskit 1.0 in-place. [Read more](https://docs.quantum.ibm.com/api/migration-guides/qiskit-1.0-installation).
```

Comment (1ucian0) [reply README.md]: does it work in PyPI?

Comment (jakelishman) [reply README.md]: I don't know. At worst it'll degrade to something that looks like

> [!WARNING] Do not try to upgrade [...]

but PyPI may support it more properly.  I couldn't immediately spot any projects doing it - Rustworkx _will_, but it hasn't released yet.

Fwiw, I imagine that more people read our README from the GitHub page than from PyPI.

Comment (1ucian0) [reply README.md]: I actually got the request because PyPI... 

Anyway, let's go that GH way and cross fingers. Somebody needs to be the first one :)

Comment (jakelishman) [reply README.md]: ah, I stand corrected then!

**Review by jakelishman — APPROVED** [review]

**Pull request merged**